projects/tensorflow/project.yaml

```diff
@@ -3,3 +3,9 @@ primary_contact: "mihaimaruseac@google.com"
 auto_ccs:
  - "dga@google.com"
  - "frankchn@google.com"
+
```

Comment:
no need of empty line, same before sanitizers.

Reply:
Done!